tests/benchmarks/test_array.py

```diff
@@ -48,35 +47,34 @@ def test_anom_mean(small_client):
     wait(anom_mean, small_client, 10 * 60)
 
 
-def test_basic_sum(small_client):
+def test_basic_sum(small_client, cluster_memory):
     # From https://github.com/dask/distributed/pull/4864
 
-    memory = cluster_memory(small_client)  # 76.66 GiB
-    target_nbytes = memory * 5
+    target_nbytes = cluster_memory * 5
     data = da.zeros(
         scaled_array_shape(target_nbytes, ("100MiB", "x")),
         chunks=(parse_bytes("100MiB") // 8, 1),
     )
-    print_size_info(memory, target_nbytes, data)
+    print_size_info(cluster_memory, target_nbytes, data)
     # 383.20 GiB - 3924 100.00 MiB chunks
 
     result = da.sum(data, axis=1)
 
     wait(result, small_client, 10 * 60)
 
 
-@pytest.mark.skip(
-    "fails in actual CI; see https://github.com/coiled/coiled-runtime/issues/253"
-)
-def test_climatic_mean(small_client):
+def test_climatic_mean(small_client, cluster_memory):
     # From https://github.com/dask/distributed/issues/2602#issuecomment-535009454
+    if cluster_memory == 1.0:
```

Comment:
Why would it ever be 1.0? The actual cluster memory would have to be 1 byte for that to happen. I assume you want to detect the not-30% case, but because you only have access to the (faked) size in bytes, not the factor, that's tricky to do. (See above for discussion.)

Reply:
I think this originates from an earlier iteration. Will need to fix that